frontend/react/src/components/layout/FormNavigation.js

```diff
@@ -2,50 +2,104 @@ import React, { Component } from "react";
 import { Button } from "@cmsgov/design-system-core";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faAngleLeft, faAngleRight } from "@fortawesome/free-solid-svg-icons";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import { selectSectionsForNav } from "../../store/selectors";
+
+const idToUrl = (id) => `/sections/${id.replace(/-/g, "/")}`;
 
 class FormNavigation extends Component {
+
   render() {
+    const { location, sections } = this.props;
```

Comment:
```suggestion
    const { history, location, sections } = this.props;
```

Pull in the history object. See the later comment about why!